### DISCUSSION TITLE
Forwarding a call in the physical telephone

### DISCUSSION THREAD

**Discussion by premiaware** [sipsorcery-org/sipsorcery#1034] [Q&A]
Let me preface this by saying that I am not a tephony expert. I need to create a 'click to call' application that selects a phone number, sends this phone number to the user's physical telephone system and automatically makes the call. I have tried several examples like SoftPhone but they behave like a phone, I only need to dial the number from software but the call is made with a physical phone in the user's desk. Where can I find an example that does this?
My application should only help the user with dialling, it should not function as a telephone.

**Comment by sipsorcery** [comment]
You need a SIP-to-PSTN gateway service to achieve this. Those services will invariably have a cost attached as they need hardware etc. to integrate.
